apis/cloud.redhat.com/v1alpha1/clowdapp_types.go

```diff
@@ -369,6 +369,9 @@ type CyndiSpec struct {
 	// +kubebuilder:validation:Pattern:="[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*"
 	AppName string `json:"appName,omitempty"`
 
+	// AdditionalFilters
+	AdditionalFilters []map[string]string `json:"additionalFilters,omitempty"`
```

Comment:
```
// +optional
// +kubebuilder:default:={}
```
Maybe?

Reply:
thought about it, prolly not need it :)

Reply:
Since the field on the ClowdApp already allows empty via the `omitempty` syntax, we decided we'd just let cyndi-operator be in charge of stating what the default value should be if the field is empty.